wrapanapi/rhevm.py

```diff
@@ -598,22 +598,23 @@ def _get_storage_domain(self, name):
         return self._get_storage_domain_service(name).get()
 
     def _get_images_service(self, storage_domain_name):
-        return self._get_storage_domain(storage_domain_name).images_service()
+        return self._get_storage_domain_service(storage_domain_name).images_service()
 
     def _get_image_service(self, storage_domain_name, image_name):
         for image in self._get_images_service(storage_domain_name).list():
             if image.name == image_name:
                 return self._get_images_service(storage_domain_name).image_service(image.id)
 
-    def import_glance_image(self, storage_domain_name, cluster_name, temp_template_name,
-                            template_name, async=True, import_as_template=True):
-        image_service = self._get_image_service(storage_domain_name, template_name)
+    def import_glance_image(self, source_storage_domain_name, source_template_name,
```

Comment:
`temp_template_name` and `template_name` were ambiguous, wanted to change them to source/target.

Second, the storage_domain_name. Still testing this one out.